src/utils/cardDefinitions.ts

```diff
@@ -0,0 +1,318 @@
+        'E7',
+        'E8',
+        'E9',
+        'E10',
+        'EJ',
+        'EQ',
+        'EK',
+        'EA',
+      ],
+      isTrump: false,
+    },
+    T: {
+      name: 'Elevation of Privilege',
+      abbreviation: 'E',
+      cards: ['T5', 'T6', 'T7', 'T8', 'T9', 'T10', 'TJ', 'TQ', 'TK', 'TA'],
+      isTrump: true,
+    },
+  },
+  [GameMode.CORNUCOPIA]: {
+    A: {
+      name: 'Data Validation & Encoding',
+      abbreviation: 'Data',
+      cards: [
+        'A2',
+        'A3',
+        'A4',
+        'A5',
+        'A6',
+        'A7',
+        'A8',
+        'A9',
+        'A10',
+        'AJ',
+        'AQ',
+        'AK',
+        'AA',
+      ],
+      isTrump: false,
+    },
+    B: {
+      name: 'Cryptography',
+      abbreviation: 'Crypt',
+      cards: [
+        'B2',
+        'B3',
+        'B4',
+        'B5',
+        'B6',
+        'B7',
+        'B8',
+        'B9',
+        'B10',
+        'BJ',
+        'BQ',
+        'BK',
+        'BA',
+      ],
+      isTrump: false,
+    },
+    C: {
+      name: 'Session Management',
+      abbreviation: 'Sessn',
+      cards: [
+        'C2',
+        'C3',
+        'C4',
+        'C5',
+        'C6',
+        'C7',
+        'C8',
+        'C9',
+        'C10',
+        'CJ',
+        'CQ',
+        'CK',
+        'CA',
+      ],
+      isTrump: false,
+    },
+    D: {
+      name: 'Authorization',
+      abbreviation: 'AuthZ',
+      cards: [
+        'D2',
+        'D3',
+        'D4',
+        'D5',
+        'D6',
+        'D7',
+        'D8',
+        'D9',
+        'D10',
+        'DJ',
+        'DQ',
+        'DK',
+        'DA',
+      ],
+      isTrump: false,
+    },
+    E: {
+      name: 'Authentication',
+      abbreviation: 'AuthN',
+      cards: [
+        'E2',
+        'E3',
+        'E4',
+        'E5',
+        'E6',
+        'E7',
+        'E8',
+        'E9',
+        'E10',
+        'EJ',
+        'EQ',
+        'EK',
+        'EA',
+      ],
+      isTrump: false,
+    },
+    T: {
+      name: 'Cornucopia',
+      abbreviation: 'Cornu',
+      cards: [
+        'T2',
+        'T3',
+        'T4',
+        'T5',
+        'T6',
+        'T7',
+        'T8',
+        'T9',
+        'T10',
+        'TJ',
+        'TQ',
+        'TK',
+        'TA',
+      ],
+      isTrump: true,
+    },
+  },
+};
+
+export function getStartingCard(gameMode: GameMode, suit: Suit): Card {
+  return CARD_DECKS[gameMode][suit].cards[0];
+}
+
+export function getSuits(gameMode: GameMode): Suit[] {
+  return Object.keys(CARD_DECKS[gameMode]) as Suit[];
+}
+
+export function getSuitDisplayName(gameMode: GameMode, suit: Suit): string {
+  return CARD_DECKS[gameMode]?.[suit]?.name ?? '';
+}
+
+function getAbbreviation(gameMode: GameMode, card: Card): string {
+  const suitHoldingCard = Object.values(CARD_DECKS[gameMode] ?? []).find(
+    (suit) => suit.cards.includes(card),
+  );
+
+  return suitHoldingCard?.abbreviation ?? '';
+}
+
+export function getCardDisplayName(
```

Comment:
The naming is a bit inconsistent here: Some other other functions also return something for a `Card` but don't have `Card` in there name.

Reply:
I think `getAbbreviation` was the only thing, not having `Card` in its name.
Will change it to `getCardAbbreviation`. Then all `Card`-related functions should have `Card` in their names.